src/init/types.ts

```diff
@@ -19,4 +19,5 @@ export type InitOptions = {
   autoRemoveUrlParams?: boolean
   loadFileSystem?: boolean
   rootKey?: string
+  useWnfs?: boolean
```

Comment:
Good question, and something that I wrestled with quite a bit. 

I'm writing up a proposal for some broader changes to initialization where this will make more sense. In short, all apps that want to use WNFS would set this option to true.

For this PR, perhaps we could do something to make it clear that apps don't need to use this option if they initialize with `permissions`. Maybe throw an exception when `useWnfs` is true and `permissions` are used?

Reply:
Yep, it will be a bit awkward at first, but it should be be backward compatible while we transition into a refactored API. Added a comment to `initialise` to document it.